tests/http/config/hello.graphql

```diff
@@ -0,0 +1,7 @@
+schema @server(port: 8080) {
+  query: Query
+}
+
+type Query {
+  hello: String! @const(data: "World!")
+}
```

Comment:
do we need this file?

Reply:
oh I accidentally committed that as well, I'll remove it